tests/python/test_assign.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+import taichi as ti
+
+
+@ti.test()
```

Comment:
Thanks! Could you write the tests in this way:

1. Make each kernel a separate test case.
2. Use 0-D Taichi fields instead of local variables `a`, `b`; then do the assertion outside the `@ti.kernel`. The reason is that assertions inside a Taichi kernel is enabled only when `init(debug=True)` 😓

Reply:
SGTM.

Reply:
BTW The code will run without any warning if `debug=True` is not set, should we drop a warning message in such condition?